ecosystem/indexer-grpc/indexer-grpc-integration-test-transaction-generator/src/main.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use aptos_indexer_grpc_server_framework::setup_logging;
+use clap::Parser;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    setup_logging(None);
+
+    let args = aptos_indexer_grpc_integration_test_transaction_generator::transaction_generator::TransactionGeneratorArgs::parse();
```

Comment:
Nit: Let's import TransactionGeneratorArgs lol, this is too long for a fully qualified usage.